lib/av_client/types.ts

```diff
@@ -271,6 +271,8 @@ export interface ElectionConfig {
   // appended data:
   affidavit: AffidavitConfig;
 
+  castRequestItemAttachmentEncryptionKey: string;
```

Comment:
The current implementation of `js-client` points to `http://dbb:3003/:election_slug/configuration` which doesn't return `castRequestItemAttachmentEncryptionKey` in response.

The `castRequestItemAttachmentEncryptionKey`  is returned only by `http://dbb:3003/:election_slug/configuration/latest_config` (see: https://github.com/aion-dk/dbb/pull/146)

@av-emil What would you like to do with it?

Reply:
@av-lukas and/or @av-mads, what's the intented difference between `dbb:3003/:election_slug/configuration` and `dbb:3003/:election_slug/configuration/latest_config`?